homeassistant/components/yale_smart_alarm/config_flow.py

```diff
@@ -66,6 +91,14 @@ async def async_step_reauth_confirm(self, user_input=None):
                     data_schema=DATA_SCHEMA,
                     errors={"base": "invalid_auth"},
                 )
+            except requests.HTTPError as error:
```

Comment:
Does the non 401 case also need to be handled here? It seems like it will fall through as a success case. I think it makes sense to add a test for this case.

Reply:
Added it in new commit

Reply:
This change isn't related to the options flow. Please remove it and create a separate PR for it.